src/main/java/mate/academy/Main.java

```diff
@@ -27,7 +35,8 @@ public static void main(String[] args) {
         secondCinemaHall.setCapacity(200);
         secondCinemaHall.setDescription("second hall with capacity 200");
 
-        CinemaHallService cinemaHallService = null;
+        CinemaHallService cinemaHallService = (CinemaHallService)
+                injector.getInstance(CinemaHallService.class);
```

Comment:
Ensure that you pass the correct class type to the injector to avoid `ClassCastException`. Double-check that `CinemaHallService.class` is the correct type for this instance.